src/main/java/com/spotinst/sdkjava/model/requests/azure/statefulNode/ApiStatefulNodeStateRequest.java

```diff
@@ -0,0 +1,19 @@
+package com.spotinst.sdkjava.model.requests.azure.statefulNode;
+
+public class ApiStatefulNodeStateRequest {
```

Comment:
Rename to ApiStatefulNodeStateChangeRequest

Reply:
Done